src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
+@EntityScan("wolox.training.models")
```

Comment:
This is unnecessary. Springboot analyzes the default locations one would expect, and you only need to use this if you have classes outside src/main/java.